### DISCUSSION TITLE
Full screen stopped working on Version Avalonia 0.10.18

### DISCUSSION THREAD

**Discussion by DenisTukhvatshin** [AvaloniaUI/Avalonia#8939] [Q&A]
Full screen stopped working on Version Avalonia 0.10.18
I did't have problem with v0.10.17.
        mc:Ignorable="d" d:DesignWidth="1024" d:DesignHeight="600"
        MinWidth="1024"
        MinHeight="600"
        WindowState="FullScreen"
        Topmost="True"
        SystemDecorations="BorderOnly"
That is my params.
Is there a solution already?

**Comment by DenisTukhvatshin** [comment]
I forgot to clarify that a resolution of 1024x768 is used

**Comment by DenisTukhvatshin** [comment]
And I use windows 10

**Comment by DenisTukhvatshin** [comment]
Here is simple example
https://github.com/DenisT90/AvaloniaStartupFullScreen0.10.18NotWorking

**Comment by timunie** [comment]
It works for me if I remove `MinWidth` and `MinHeight`.

**Comment by DenisTukhvatshin** [comment]
Why it doesn't work with that params MinWidth and MinHeight ?

**Reply by timunie** [reply]
I'm not sure. Probably it's a bug. In that case you may open a bug report.

**Comment by DenisTukhvatshin** [comment]
How to open a discussion about this error?

**Reply by timunie** [reply]
https://github.com/AvaloniaUI/Avalonia/issues

**Reply by DenisTukhvatshin** [reply]
https://github.com/AvaloniaUI/Avalonia/issues/8975
created
